pincer/objects/guild/guild.py

```diff
@@ -421,9 +421,10 @@ async def from_id(cls, client: Client, _id: Union[int, Snowflake]) -> Guild:
     async def get_member(self, _id: int) -> GuildMember:
         """|coro|
         Fetches a GuildMember from its identifier
+
         Parameters
         ----------
-        _id:
+        _id: int
```

Comment:
Shouldn't it be `` :class:`int` ``?

Reply:
It works and is more concise 
https://pincer-test.readthedocs.io/en/fix-docs/pincer.objects.guild.html#pincer.objects.Guild.get_member